src/function/response.ts

```diff
@@ -12,3 +12,7 @@ export interface Response {
 export interface BuilderResponse extends Response {
   ttl?: number
 }
+
+export interface StreamingResponse extends Omit<Response, 'body'> {
+  body?: string | NodeJS.ReadableStream
```

Comment:
In the future, we should also support Web Streams here (https://nodejs.org/api/webstreams.html). But they're still Experimental in Node 19, so I *think* we should hold out on this for now? OTOH, the interface is based on the stable WHATWG spec, so i'm not sure if we can expect any breaking changes as this stabilizes ...

Reply:
ha, there's a way we can make both work right now without having to think about stability! [eb1baed](https://github.com/netlify/functions/pull/395/commits/eb1baed44e27b5b608f9594d9b74abfce48721b6)

noice